content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -324,6 +324,43 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Pod failure policy
+
+{{< note >}}
+This feature is introduced as an Alpha feature controller by two feature gates
+`PodDisruptionConditions` and `JobPodFailurePolicy`, both available in Kubernetes v1.25.
+`PodDisruptionConditions` enables
+{{< /note >}}
+
+Example job configuration using this feature:
+
+```yaml
+apiVersion: v1
+kind: Job
+spec:
+  template:
+    spec:
+      containers:
+      - name: job-container
+        image: job-image
+        command: ["./program"]
+  backoffLimit: 6
+  podFailurePolicy:
+    rules:
+    - action: Ignore
+      onPodConditions:
+      - type: DisruptionTarget
+    - action: FailJob
+      onExitCodes:
+        operator: In
+        values: [40,41,42]
+```
+
+{{< note >}}
+The `.spec.podFailurePolicy` field in JobSpec API can only be used in combination
+with `.spec.restartPolicy=Never`.
```

Comment:
```suggestion
with `.spec.restartPolicy` set to `Never`.
```

This avoids switching between YAML style (`restartPolicy: Never`) and a different style.